Jenkinsfile

```diff
@@ -434,6 +471,10 @@ pipeline {
             name: "OPENCL_BACKEND",
             defaultValue: false,
             description: "Enable OpenCL backend stages")
+        booleanParam(
+            name: "PERF_TEST",
```

Comment:
Let's remain consistent. Please rename this to `BUILD_PERFORMANCE` (or `BUILD_PERFORMANCE_TEST` if you wish) and move near definition of BUILD_PACKAGED.

Reply:
I consider this as resolved.